nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -97,6 +97,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - Calling `makeSetupHook` without passing a `name` argument is deprecated.
 
+- Texinfo has been updated to 7.0.2
+
```

Comment:
We should expand this with more information especially why this is relevant or drop it.

Reply:
I thought that a major version change would merit an entry in the release notes. Perusing quickly the texinfo [news](https://git.savannah.gnu.org/cgit/texinfo.git/plain/NEWS) and [ChangeLog](https://git.savannah.gnu.org/cgit/texinfo.git/plain/ChangeLog) I see no obvious mention of breaking changes.

Is there anything in the [news](https://git.savannah.gnu.org/cgit/texinfo.git/plain/NEWS) or [ChangeLog](https://git.savannah.gnu.org/cgit/texinfo.git/plain/ChangeLog) that strikes you as noteworthy for the 2305 release notes, @SuperSandro2000?

Reply:
Texinfo is a dependency of many softwares, and it is a major version bump.

Reply:
The question is whether this is actually backward incompatibile change or would be better placed in the **Other Notable Changes** or possibly even **Highlights** section?

Reply:
I can't see anything in the changelog that would warrant a release notes entry, let alone a highlight. Keep in mind that the release notes are intended for NixOS end users, to tell them how their system might change and how to adapt. Texinfo having a major bump does not seem like something users should care about.

Reply:
I'm inclined to agree with @ncfavier here, but cc @mweinelt @Mic92

Reply:
Well, if this is not a breaking change, I vote for remove this notice.

Reply:
It seems more folks are in favor of removing this entry to the release notes, hence it has been removed. Thanks for chiming in everyone, much appreciated!